src/newSeed/baseStage.scss

```diff
@@ -518,21 +518,17 @@
 
           .tokenInfo {
             display: grid;
-            grid-template-columns: 1fr 0.5fr 0.5fr;
+            grid-template-columns: repeat(3, auto);
+            column-gap: 10px;
+            align-items: center;
             width: fit-content;
             & div {
               align-self: center;
-              margin-right: 0;
             }
             & > img {
               @include tokenIcon;
               height: 19px;
             }
-            &.projectLogo {
```

Comment:
1) the class `tokenInfo` is used in 8 places on this page.  Can you please make sure that the styles are applied properly for all cases?  Rename the styles if you need to.

2) please make sure that the top-level divs under all of the `tokenInfo` have their own proper class, and in the scss under `tokenInfo`, remove the use of `& div`, referring to a proper class instead.

3) can you please add a bit more space here between clipboard button and symbol?  Like this.  Note you can select `.etherscanlink` if needed.

![image](https://user-images.githubusercontent.com/1821666/125653615-4383e1b8-7745-48b2-b642-1b3773db917a.png)

Reply:
1. I've controlled it again for all 8 instances of `tokenInfo`.
2. The declaration for the nested `div` is now removed, as it didn't have any effect, hence no need for a new class name.
3. I've added 10px right-margin for `.etherscanlink` if it nested inside `.tokenInfo`.